pages/common/tmux.md

```diff
@@ -24,7 +24,7 @@
 
 - Detach from session:
 
-`Ctrl + B, D`
+`Ctrl + B, d`
```

Comment:
Since we use `Ctrl +` here, would it not be safe to assume that the `Ctrl + B` is `Ctrl + B` and not `Ctrl + Shift + B`? The letters on my keyboard, at least, are in uppercase. It's a shame markdown doesn't have a `<kbd>` tag equivalent.

In the same way, a single `D` should mean the `D` on your keyboard, and not `SHIFT + D`.